addons/sale_coupon/tests/test_program_numbers.py

```diff
@@ -1146,3 +1146,53 @@ def test_program_modify_reward_line_qty(self):
         self.assertEqual(len(order.order_line), 2, 'The order must contain 2 order lines: 1x Product F and 1x 5$ discount')
         self.assertEqual(order.amount_total, 190.0, 'The price must be 190.0 since there is now 2x 5$ discount and 2x Product F')
         self.assertEqual(order.order_line.filtered(lambda x: x.is_reward_line).price_unit, -5, 'The discount unit price should still be -5 after the quantity was manually changed')
+
+    def test_program_multiple_promotions(self):
```

Comment:
If you don't mind, could you use a more explicit name please?
Something indicating it test the maximum_use_number behavior, eg `test_program_maximum_use_number` or `test_program_maximum_use_number_with_other_promo` (I don't mind having long function name, as it help figuring what will the test do)